dataeng/resources/run-course-exporter.sh

```diff
@@ -10,7 +10,7 @@ pip install mysql-connector-python -e .
 popd
 
 # Get name of other (platform) virtual environment
-source platform_venv
+source platform_venv_path
```

Comment:
I thought the course exporter uses Python 2 or "legacy" still? As defined here?:

https://github.com/edx/jenkins-job-dsl/blob/master/dataeng/resources/setup-platform-venv-legacy.sh#L23-L22

This change implies it uses this setup instead?:

https://github.com/edx/jenkins-job-dsl/blob/master/dataeng/resources/setup-platform-venv-py3.sh#L21

Reply:
There are 2 virtualenvs at play.  The platform one is python 3, and the exporter one is python 2.  This line is sourcing the file containing the path to the python 3 platform venv, but this script `run-course-exporter.sh` is running inside a python 2 venv.  Is that confusing enough?

Reply:
The setup-platform-venv-*.sh files are for creating the platform virtualenv, and this `run-course-exporter.sh` file creates the exporter py2 venv using shiningpanda (i.e. virtualenv build step).